rosidl_parser/rosidl_parser/parser.py

```diff
@@ -234,7 +234,13 @@ def extract_content_from_ast(tree):
                 namespaces=goal_request.structure.type.namespaces,
                 name=goal_request_basename),
             goal_request, result_response, feedback_message)
-        content.elements.append(action)
+
+        all_includes = content.get_elements_of_type(Include)
+        unique_include_locators = set([
+            include.locator for include in all_includes])
```

Comment:
I would expect the linter to warn about this. This likely needs to be a set comprehension instead.

Reply:
@hidmic Once you have checked this (and updated if necessary) please squash merge this PR into `idl-stage-7`.